hokusai/production.yml

```diff
@@ -62,15 +62,14 @@ metadata:
     app: doppler
     component: web
     layer: application
-  name: doppler-web
+  name: doppler-web-internal
   namespace: default
 spec:
   ports:
-    - port: 80
+    - port: 8080
       protocol: TCP
       name: http
       targetPort: 8080
-      nodePort: null
```

Comment:
If we create a fresh ClusterIP service - NodePort will never be assigned so we won't need to add `nodePort: null` to downgrade it